### DISCUSSION TITLE
Where do I set up the source model?

### DISCUSSION THREAD

**Discussion by CHsCCHHSS** [bmaltais/kohya_ss#2512] [Q&A]
I'm a new user. I'm following tutorials. The tutorial tells LORA to choose a source model, but I can't find it in my program.
I don't have a menu to choose a sauce model. Where is it?
In the video or picture, the source model menu appears on the LORA tab, 
but in the version I downloaded a few days ago, there is no source menu.

**Comment by 5KilosOfCheese** [comment]
![kuva](https://github.com/bmaltais/kohya_ss/assets/117771081/157158dd-8efc-4dae-ab87-a81365b30377)
If you choose a model from the dropdown menu, the system will download it. Alternatively you can navigate using the folder icon to the folder in to which you have downloaded a model and choose that.
